e2e/package.json

```diff
@@ -5,7 +5,8 @@
     "ios": "react-native run-ios",
     "web": "expo start --web",
     "start": "expo start --dev-client",
-    "bundle:ios": "react-native bundle --entry-file index.js --bundle-output ios/main.jsbundle --dev false --platform ios --assets-dest ios"
+    "bundle:ios": "react-native bundle --entry-file index.js --bundle-output ios/main.jsbundle --dev false --platform ios --assets-dest ios",
+    "bundle:android": "mkdir -p android/app/src/main/assets; react-native bundle --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --dev false --platform android --assets-dest android/app/src/main/res"
```

Comment:
bundle:android command isn't currently used anywhere, but it will be used in the CI